precode.sh

```diff
@@ -6,38 +6,42 @@
 #   dir2
 #   dir3
 #       dir4
-
+mkdir -p task task/dir1 task/dir2 task/dir3/dir4
```

Comment:
За счет использования флага `-p` можно убрать
```suggestion
mkdir -p task/dir1 task/dir2 task/dir3/dir4
```